pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/template/Template.java

```diff
@@ -88,12 +88,19 @@ public String getTemplateText() {
 
     public void addVariable(TemplateVariable variable) {
         if (variable != null) {
+            if (variables == null) {
+                variables = new ArrayList<>();
+            }
```

Comment:
Add this check in order to avoid NPE (in case when an empty "variables":[]" is not declared in conf).